.coveragerc

```diff
@@ -46,6 +49,13 @@ exclude_lines =
     # Decorators
```

Comment:
Comment here: `return NotImplemented` should go off the list, since it's part of actual functionality.

Reply:
Instead, we may think about adding the `RuntimeError` stuff that's just defensive coding.